src/resources/Incorporations/benefitCompany.ts

```diff
@@ -39,6 +39,16 @@ export const BenefitCompanyResource: IncorporationResourceIF = {
   shareClasses: {
     countMinimum: 1
   },
+  incorporationArticles: {
+    articles: 'Benefit Company Articles',
+    articlesTooltip: 'The Articles for a Benefit Company must state the benefits the company intends to provide to ' +
+     'society, as well as outlining the rules and procedures for corporate matters such as holding meetings, issuing' +
+     ' and transferring shares, and duties of directors and officers.',
```

Comment:
Move leading space to end of previous line (to be consistent with all other multi lines). Thx.